app/containers/App/Sidebar/Sidebar.jsx

```diff
@@ -16,110 +16,117 @@ import TokenSaleIcon from '../../../assets/navigation/tokens.svg'
 import SettingsIcon from '../../../assets/navigation/settings.svg'
 import { ROUTES } from '../../../core/constants'
 
-import LogoWithoutText from '../../../assets/images/logo-without-text-black.png'
+import LightLogoWithoutText from '../../../assets/images/logo-without-text-black.png'
+import DarkLogoWithoutText from '../../../assets/images/logo-without-text.png'
 
 type Props = {
-  className: string
+  className: string,
+  theme: 'Light' | 'Dark'
```

Comment:
I think its best if the theme prop will have a `ThemeType` (this type can be exported from where the theme was defined)

```
const THEME = {
 LIGHT: 'Light',
 DARK: 'Dark'
}

type ThemeType = 'Light' | 'Dark'
```